src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  position: relative;
+  margin: 20px auto;
+}
+
+.search-icon {
+  position: absolute;
```

Comment:
don't move position out of other styles, it's ok to repeat writing it

Reply:
Thanks. I'll take it into account.